src/gui/menu_list.cpp

```diff
@@ -0,0 +1,23 @@
+#include "gui/menu_item.hpp"
+#include "gui/menu_list.hpp"
+#include "gui/menu_manager.hpp"
+
+ListMenu::ListMenu(const std::vector<std::string>& items_, int* selected_) : 
+  selected(selected_)
+{
+  for(int i = 0; i < items_.size(); i++) {
+    add_entry(i, items_[i]);
```

Comment:
I missed that one - member variables should always be prefixed with `m_` per the [coding style](https://github.com/SuperTux/supertux/blob/master/CODINGSTYLE.md), so after that there's no need for trailing underscores for the argument names in the constructor.

(`items` also no longer has a member variable with the same name :D)